content/en/tags-type.md

```diff
@@ -15,257 +15,17 @@ related:
 
 ## Overview
 
-The @type tag allows you to provide a type expression identifying the type of value that a symbol
-may contain, or the type of value returned by a function. You can also include type expressions with
-many other JSDoc tags, such as the [@param tag][param-tag].
+The `@type` tag allows you to provide a type expression identifying the type of value that a symbol
+may contain, or inline to indicate that a symbol is of a specific type.
```

Comment:
I found this confusing. Please rewrite along these lines: "...may contain. For function parameters, you can also use an inline `@type` tag to identify a parameter's type."

Reply:
This text works with my casting example, below, hence why it might be confusing. I've taken the suggestion though and made it about function parameters.